### PR TITLE
Ft test coverage#187570807

### PR DESCRIPTION
**Pull request by Ishimwe7**
### What does this PR do?
This Pull Request serves to setup testing environment for our components
### Description of Task to be completed?
1. Setting Jest testing environment.
2. Running some sample tests to see if they can work
### How should this be manually tested?
Inside root directory run **npm run test** and **npm run test:coverage** if you want to see coverage

### CODE REVIEW

File: coverage/lcov-report/block-navigation.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+var jumpToCode = (function init() {
+    // Classes of code we would like to highlight in the file view
```
Comment (kirengamartial) [inline coverage/lcov-report/block-navigation.js:3]: remove those comments they are too many

**Review by kirengamartial — CHANGES_REQUESTED** [review]
consider the review i gave you and if those photos are not being used in our application you may remove them

**Review by rodriguecyber — APPROVED** [review]

**Review by robsdagreat — APPROVED** [review]
Everything is good but consider using yarn

File: coverage/lcov-report/block-navigation.js (hunk 1 above)
Comment (Ishimwe7) [reply coverage/lcov-report/block-navigation.js:3]: I think it is not good practice to edit auto-generates files. I didn't write them though.

**Pull request closed without merging**